csrankings.csv

```diff
@@ -5572,6 +5573,7 @@ Hong Mei,Peking University,http://sei.pku.edu.cn/~meih/index_en.html,b-ZYIwoAAAA
 Hong Q. Ding,University of Texas at Arlington,http://ranger.uta.edu/~chqding,q7FfnjgAAAAJ
 Hong Qin,Stony Brook University,http://www3.cs.stonybrook.edu/~qin,Wpsc65EAAAAJ
 Hong Shen 0001,University of Adelaide,https://cs.adelaide.edu.au/~hong,tBnVrjEAAAAJ
+Hong Yu,University of Massachusetts Lowell,http://bio-nlp.org/,TyXe64wAAAAJ

```

Comment:
AFAIK Hong is at UMass Med.

Reply:
(and thus cannot be listed for UML)

Reply:
She moved to UML in Jan 2018.

https://www.uml.edu/Sciences/computer-science/faculty/Yu-Hong.aspx